apps/app/src/features/search/client/components/SearchModal.tsx

```diff
@@ -49,10 +49,13 @@ const SearchModal = (): JSX.Element => {
   };
 
   useEffect(() => {
-    if (!searchModalData?.isOpened) {
+    if (!searchModalData?.isOpened || typeof searchModalData?.searchKeyword === 'undefined') {
```

Comment:
null チェックに変更する
https://tips.weseek.co.jp/5fb74e7aa1ba4200489a4bd1#%E5%88%A4%E5%AE%9A%E5%BC%8F
```suggestion
    if (searchModalData?.searchKeyword != null)
```

Reply:
修正しました。